apps/total-typescript/src/components/exercise-overlay.tsx

```diff
@@ -401,7 +404,39 @@ const BlockedOverlay: React.FC = () => {
   )
 }
 
-export {ExerciseOverlay, DefaultOverlay, FinishedOverlay, BlockedOverlay}
+type LoadingOverlayProps = {}
+
+const LoadingOverlay: React.FC<LoadingOverlayProps> = () => {
+  const {
+    muxPlayerProps: {playbackId},
+  } = useMuxPlayer()
+  const thumbnail = `https://image.mux.com/${playbackId}/thumbnail.png?width=480&height=384&fit_mode=preserve`
```

Comment:
thumbnail needs `playbackId`. we can get away with relatively small image to improve loading speed by blurring it with css.